packages/data-explorer-ui/src/hooks/useWindowResize.ts

```diff
@@ -7,9 +7,10 @@ export interface WindowSize {
 
 /**
  * Listens to window resize event and returns window height and width values.
+ * @param timeout - Timeout in milliseconds to wait before recalculating window size.
  * @returns window height and width values.
  */
-export const useWindowResize = (): WindowSize => {
+export const useWindowResize = (timeout = 200): WindowSize => {
```

Comment:
Added timeout to the `useWindowResize` hook... we may wish to increase or decrease this value depending on usage.